clang/test/Driver/sycl-offload-arch-amd-gpu-cpu.cpp

```diff
@@ -91,15 +88,6 @@
 
 // AMD CPUs
 
-// RUN: %clangxx -### --offload-new-driver -fsycl --offload-arch=gfx600 -nogpulib -fno-sycl-libspirv %s 2>&1 | \
-// RUN:   FileCheck %s --check-prefixes=TARGET-TRIPLE-AMD-CPU,CLANG-OFFLOAD-PACKAGER-AMD-CPU -DDEV_STR=gfx600
-
-// RUN: %clangxx -### --offload-new-driver -fsycl --offload-arch=gfx601 -nogpulib -fno-sycl-libspirv %s 2>&1 | \
-// RUN:   FileCheck %s --check-prefixes=TARGET-TRIPLE-AMD-CPU,CLANG-OFFLOAD-PACKAGER-AMD-CPU -DDEV_STR=gfx601
-
-// RUN: %clangxx -### --offload-new-driver -fsycl --offload-arch=gfx602 -nogpulib -fno-sycl-libspirv %s 2>&1 | \
-// RUN:   FileCheck %s --check-prefixes=TARGET-TRIPLE-AMD-CPU,CLANG-OFFLOAD-PACKAGER-AMD-CPU -DDEV_STR=gfx602
```

Comment:
Why are all of these being cleaned out - are they not supported?

Reply:
yes, these are not supported AMD targets.